projects/js-packages/charts/package.json

```diff
@@ -17,15 +17,33 @@
 		"test": "jest --config=tests/jest.config.cjs",
 		"test-coverage": "pnpm run test --coverage"
 	},
+	"dependencies": {
+		"@visx/axis": "^3.8.0",
+		"@visx/group": "^3.8.0",
+		"@visx/scale": "^3.8.0",
+		"@visx/shape": "^3.8.0"
+	},
 	"devDependencies": {
 		"jest": "29.7.0",
 		"jest-environment-jsdom": "29.7.0",
 		"jest-extended": "4.0.2",
-		"typescript": "5.0.4"
+		"typescript": "5.0.4",
+		"@types/react": "18.3.12",
+		"@types/react-dom": "18.3.1",
+		"react": "18.3.1",
+		"react-dom": "18.3.1",
```

Comment:
If you don't consume a dependency directly in the package, it's fine to use it only as a peer dependency.

Reply:
So, you can remove both `react` and `react-dom` from `devDependencies` and keep them only in `peerDependencies`